examples/step-19/step-19.cc

```diff
@@ -650,6 +651,8 @@ namespace Step19
   {
     const double dt = time.get_next_step_size();
 
+    std::vector<double>       solution_values(fe.n_dofs_per_cell());
+    FEPointEvaluation<1, dim> evaluator(mapping, fe);
```

Comment:
Can you change this to the new interface we have on master? Then this PR should be good to be merged.

Reply:
I am a bit confused. Isn't this the new interface? It already uses `EvaluationFlags::gradients`.

Reply:
I guess you mean that I should rebase so that the changes to include/deal.II/fe/fe_point_evaluation.h are not visible?

Reply:
Sorry, yes, I misread. I was thinking about the changes in the interface for the evaluation.